ui/marketplace/MarketplaceAppTopBar.tsx

```diff
@@ -0,0 +1,106 @@
+import { chakra, Flex, Tooltip, Skeleton, Box } from '@chakra-ui/react';
+import React from 'react';
+
+import type { MarketplaceAppOverview } from 'types/client/marketplace';
+
+import { route } from 'nextjs-routes';
+
+import { useAppContext } from 'lib/contexts/app';
+import type { IconName } from 'ui/shared/IconSvg';
+import IconSvg from 'ui/shared/IconSvg';
+import LinkExternal from 'ui/shared/LinkExternal';
+import LinkInternal from 'ui/shared/LinkInternal';
+
+import MarketplaceAppInfo from './MarketplaceAppInfo';
+
+type Props = {
+  isWalletConnected: boolean;
+  data: MarketplaceAppOverview | undefined;
+  isPending: boolean;
```

Comment:
We typically use `isLoading` props throughout the app to display the loading state of components. It's best to maintain this naming convention.